WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginEpilogueActivity.java

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.accounts;
+
+import org.wordpress.android.R;
+import org.wordpress.android.util.ToastUtils;
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+
+public class LoginEpilogueActivity extends AppCompatActivity implements LoginEpilogueFragment.LoginEpilogueListener {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.login_epilogue_activity);
+
+        if (savedInstanceState == null) {
```

Comment:
We don't _need_ a fragment per se in this case since the parent Activity only utilises this single fragment. The epilogue screen didn't start it's life in a dedicated Activity though but rather, as just another screen under `LoginActivity.java`. It might still change parents down the road as the whole feature is developed. Keeping it as a fragment serves the purpose of easy movement between parents.